examples/clamped_truss/own_weight_clamped_truss.jl

```diff
@@ -0,0 +1,117 @@
+# ---------------------------------
+# Own weight clamped truss example
+# --------------------------------
+#=
+This model consist of a clamped truss with its own weight. First the
+weight is applied and consecutive the load at the time starting from the own
+weight deformed configuration.
+
+# Analysis 1: Own weight
+
+## Forces
+-> g
+--------> F
+
+# Analysis 1: Own weight
+#-----------------------------
+# ->  ->  ->  ->  ->  ->  -> -
+#-----------------------------
+
+# Analysis 2: Own weight + load
+#-----------------------------
+# ->  ->  ->  ->  ->  ->  -> - -----> F
+#-----------------------------
+=#
+using ONSAS, Test
+using Dictionaries: dictionary
+
+"Return problem parameters"
+function parameters()
+    N = 10     # Number of elements.
+    E = 30e6    # Young's modulus.
+    ν = 0.3     # Poisson's ratio.
+    ρ = 750.0   # Density.
+    L = 200     # Element length.
+    A = 1       # Cross section area.
+    F = 10e6    # Force at the tip
+    g = 9.81    # Gravity
+    ϵ_model = RotatedEngineeringStrain
+    g, N, E, ν, ρ, L, A, F, ϵ_model
```

Comment:
in a case like this you may prefer to return a named tuple, so `(; g, N, ..)` then unpack it at the callsite in the same way `(; g, N, ...) = ...`. This prevents from potential bugs when reordering variables or removing / adding new ones.

Reply:
Done